calc.js

```diff
@@ -0,0 +1,502 @@
+      } else {
+        screen.innerHTML = Math.cos(screen.innerHTML).toPrecision(10);
+      }
+      break;
+    case "tan":
+      if (flag) {
+        screen.innerHTML = Math.tan(
+          (screen.innerHTML * Math.PI) / 180
+        ).toPrecision(2);
+      } else {
+        screen.innerHTML = Math.tan(screen.innerHTML).toPrecision(10);
+      }
+      break;
+    case "sec":
+      if (flag) {
+        screen.innerHTML = Math.sec(
+          (screen.innerHTML * Math.PI) / 180
+        ).toPrecision(2);
+      } else {
+        screen.innerHTML = Math.sec(screen.innerHTML).toPrecision(10);
+      }
+      break;
+    case "ceil":
+      screen.innerHTML = Math.ceil(screen.innerHTML);
+      break;
+    case "floor":
+      screen.innerHTML = Math.floor(screen.innerHTML);
+      break;
+    case "round":
+      screen.innerHTML = Math.round(screen.innerHTML);
+      break;
+    case "f-e":
+      screen.innerHTML = Math.pow(10, screen.innerHTML);
+      break;
+    case "backspace":
+      screen.innerHTML = backSpace(screen.innerHTML);
+      break;
+    case "clear":
+      //clear the screen
+      screen.innerHTML = "";
+      break;
+    case "pi":
+      if (screen.innerHTML === " ") {
+        screen.innerHTML = Math.PI;
+      } else {
+        screen.innerHTML = screen.innerHTML * Math.PI;
+      }
+      break;
+    case "equal":
+      if (screen.innerHTML.includes("^")) {
+        screen.innerHTML = root(screen.innerHTML);
+      } else {
+        screen.innerHTML = evaluate(screen.innerHTML);
+      }
+      break;
+    case "pos-neg":
+      if (screen.innerHTML === " ") {
+        alert("Enter the number you want");
+      } else {
+        screen.innerHTML = changeSign(screen.innerHTML);
+      }
+      break;
+    case "exponential":
+      if (screen.innerHTML === "") {
+        screen.innerHTML = Math.E;
+        console.log(screen.innerHTML);
+      } else {
+        screen.innerHTML = screen.innerHTML * Math.E;
+      }
+      break;
+    case "pow2":
+      screen.innerHTML = Math.pow(screen.innerHTML, 2);
+      break;
+    case "denom":
+      screen.innerHTML = 1 / screen.innerHTML;
+      break;
+    case "absolute":
+      screen.innerHTML = Math.abs(screen.innerHTML);
+      break;
+    case "root":
+      screen.innerHTML = Math.sqrt(screen.innerHTML);
+      break;
+    case "open-parenthesis":
+      screen.innerHTML = screen.innerHTML + "(";
+      break;
+    case "close-parenthesis":
+      screen.innerHTML = screen.innerHTML + ")";
+      break;
+    case "power":
+      screen.innerHTML = screen.innerHTML + "^";
+      break;
+    case "exponent":
+      screen.innerHTML = Math.exp(screen.innerHTML);
+      break;
+    case "mod":
+      screen.innerHTML = Math.abs(screen.innerHTML);
+      break;
+    case "base10":
+      screen.innerHTML = Math.pow(10, screen.innerHTML);
+      break;
+    case "logarithm":
+      if (screen.innerHTML === "") {
+        alert("Enter the number you want");
```

Comment:
make alert message appropriate. Don't use same alert message each place.. be like "First enter number for log"